game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -0,0 +1,47 @@
+package io.aigar.game
+
+import com.github.jpbetz.subspace._
+
+object Cell {
+  /**
+   * Force (in Newtons) applied when moving.
+   */
+  final val MovementForce = 10f
+
+  /**
+   * Default mass of a cell.
+   */
+  final val DefaultMass = 1 // TODO determine this once we have visualization/etc.
+}
+
+class Cell {
+  var position = new Vector2(0f, 0f)
+  var target = new Vector2(0f, 0f)
+  var mass = Cell.DefaultMass
+  private var _velocity = new Vector2(0f, 0f)
+
+  /**
+   * The maximum speed (length of the velocity) for the cell, in units per
+   * second.
+   */
+  def maxSpeed = {
+    15f //TODO depend on mass?
+  }
+
+  def velocity = _velocity
```

Comment:
I was that scala had something like C# where you can define the attribute, getter and setter really easily. Still not too bad :)